packages/web/src/apollo/index.tsx

```diff
@@ -76,7 +75,7 @@ export const RedwoodApolloProvider: React.FunctionComponent<{
     <FetchConfigProvider useAuth={useAuth}>
       <ApolloProviderWithFetchConfig config={graphQLClientConfig}>
         <GraphQLHooksProvider useQuery={useQuery} useMutation={useMutation}>
-          <FlashProvider>{children}</FlashProvider>
+          {children}
```

Comment:
Ooh, I didn't realize that we didn't have to place `<Toaster />` here. This means that only people using the scaffold stuff need to use toaster. I then think we should place these in `@redwoodjs/web/flash`.

I can do this for you in this PR if you agree.

Reply:
Sounds good to me! Yeah right now I include <Toaster> in the scaffold layout so that it’s only contained within your generated stuff, in case you don’t end up using it in your main app. 

Yesterday I saw a recommendation in the docs that you only have one instance of <Toaster> in your app...I haven’t tried having two scaffolds, for example, and moving between them. It might just get confused if you have a toast showing, then move to another instance of <Toaster> and show a second notification...I’ll test on Monday.

Reply:
Okay, two separate scaffolds with two separate `<Toaster />`s in their layouts aren't causing any issues for me!

You want to go ahead and move that to `@redwoodjs/web/flash` @peterp? Wondering if we should rename it now. I only used "flash" because that's what Rails calls it, but now that it's using `react-hot-toast` maybe we should just make it `web/toast`?

Reply:
I will get this done tomorrow, pinky promise.

Reply:
I like `web/toast`